libs/MotionKit/tests/PID_test.cpp

```diff
@@ -0,0 +1,85 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "PID.h"
+
+#include "gtest/gtest.h"
+
+using namespace leka;
+
+class PIDTest : public ::testing::Test
+{
+  protected:
+	PIDTest() = default;
+
+	// void SetUp() override { }
+	// void TearDown() override {}
+
+	PID pid {};
+};
+
+TEST_F(PIDTest, initialization)
+{
+	ASSERT_NE(&pid, nullptr);
+}
+
+TEST_F(PIDTest, processPIDDefaultPosition)
+{
+	auto pitch = 0.F;
+	auto roll  = 0.F;
+	auto yaw   = 180.F;
+
+	auto [speed, direction] = pid.processPID(pitch, roll, yaw);
+
+	EXPECT_EQ(speed, 0.F);
+	EXPECT_EQ(direction, Rotation::clockwise);
+}
+
+TEST_F(PIDTest, processPIDRolledOverAHalfRight)
+{
+	auto pitch = 0.F;
+	auto roll  = 0.F;
+	auto yaw   = 0.F;
+
+	auto [speed, direction] = pid.processPID(pitch, roll, yaw);
+
+	EXPECT_EQ(speed, 1.8F);
```

Comment:
`1.8F` appears like a magic number, can you precise how did you come to this value?

Reply:
You can set it as `max_speed_value` and explain in comment why this is the maximal value, and not `1.0F` or `2.0F`